source/documentation/guidance/pentest.md

```diff
@@ -13,19 +13,10 @@ You must send the below information to [gov-uk-paas-support@digital.cabinet-offi
 |Peak bandwidth that the tests will consume in gigabits per second|
 |Peak number of requests per second the tests will perform|
 |Start and end dates and times of the test in the format YYYY-MM-DDTHH:MM:SSZ for example 2017-09-26T09:00:00Z|
-|The metrics you will measure to decide if the test succeeds or fails|
 |Is it possible to stop the test immediately if there is an issue?|
 
-|Supplied by tenant|
-|:---|
-|Is the tester a third-party company?|
-|What is the purpose of the test and what is being covered?|
```

Comment:
I disagree with removing this, if we can spot patterns then there may be an way we can reduce the need for the tests

Reply:
My assumption is that everyone will just say the same thing - "I need to do a pen test because it's required by my organisation" or similar. We even say (slightly higher up) "It is your responsibility to penetration test your app to make sure it’s secure"...

Reply:
OK fair enough